### ISSUE TITLE
[Question] PIXI V4?

### ISSUE THREAD

**Issue by alextappin** [georgiee/lab-pixicam#5]
Looking to use this with a current PIXI project I am doing at work.

Will this project work with PIXI v4? I notice you are using v3 in your examples.

**Comment by georgiee** [comment]
Hello Alex,
I did not test it yet with any version of v4. I doubt it as the internal structure might have changed dramatically. When I have time to dig deeper through v4 I might update the library appropriately.

**Comment by navigator117** [comment]
Yes, it's not work with pixi v4, please fix it. Change camera position not get view update, may be help :)
